### PR TITLE
Responsive ui - round 3

### PR DESCRIPTION
**Pull request by beschoenen**

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by beschoenen** [comment]
@neoatomic Can I get your opinion on the Font-Awesome icons.

**Comment by neoatomic** [comment]
Another round.? :P 

Could you add a few quick screenshots ? (its a little messy to checkout commits on my Synology)

**Comment by beschoenen** [comment]
Why not ;D (Add show still isn't responsive anyway :x)

image

**Comment by neoatomic** [comment]
Personally i like colored icons, but its not a biggy if you want to change them.
(i know @miigotu prefers the black/white once's.)

However the issue you are going to run into, are the icons for functions like KODI, Plex etc. 
A default template of icons doesn't have those. And in the past we ended up with not fitting icons for functions, or the same icon for multiple functions. Thats the limitation of a default icon template.
So thats the main concern i have here.

**Comment by miigotu** [comment]
The colored icons were pretty well received by the community. I originally didnt like them but they have grown on me. We used to use the bootstrap/fa icons.

**Comment by neoatomic** [comment]
Lets stick with the colored icons then. :+1:

**Comment by gaetan-petit** [comment]
White icons are so much better on every levels !

**Comment by miigotu** [comment]
That's racist =p

**Comment by beschoenen** [comment]
I've been a bit non-existent lately, I'm hoping it'll change in a few weeks when summer vacation starts.

**Comment by miigotu** [comment]
No worries, SR isnt in a consistent state of disrepair for once.

**Comment by mcactive** [comment]
@beschoenen if you need any help with this, let me know.. My Python skills aren't that good, but HTML, CSS, JS are perfect and I'm happy to help!

File: gui/slick/views/home_postprocess.mako [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
+                        <b class="pull-lg-right pull-md-right pull-sm-right">${_('Enter the folder containing the episode')}</b>
                     </div>
                     <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12">
                         <input type="text" name="proc_dir" id="episodeDir" class="form-control form-control-inline input-sm" autocapitalize="off" title="directory"/>
                     </div>
                 </div>
                 <div class="row">
                     <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12">
-                        <b class="pull-lg-right pull-md-right">${_('Process Method to be used')}</b>
+                        <b class="pull-lg-right pull-md-right pull-sm-right">${_('Process Method to be used')}</b>
```
Comment (miigotu) [inline gui/slick/views/home_postprocess.mako:25]: wouldnt a plain pull-right be the same as using sm, md, and lg?

Comment (beschoenen) [inline gui/slick/views/home_postprocess.mako:25]: No, this is a `float: right` within the column `div`

**Comment by miigotu** [comment]
I think the FA should be handled by grunt/bower, other than that whats the status?

**Comment by beschoenen** [comment]
@miigotu 
Sorry, FA?
Status of? The PR?

**Comment by miigotu** [comment]
FA=font-awesome, and yes the status of the PR xD

**Comment by beschoenen** [comment]
Yea, font-awesome could be moved to bower, not sure how tho, I'd appreciate it if you could add it.
Also, do you want to keep the white icons otherwise FA wouldn't be needed?

The PR can be merged at any time, I'm just doing small stuff.
I'll push little stuff like this straight to develop from now on if that's okay.

**Comment by miigotu** [comment]
We generally dont push anything directly to develop, since develop has to be in a constant working state (and then we find the bugs there before release)

**Pull request merged**